services/web/server/tests/unit/with_dbs/02/test_projects_handlers__open_close.py

```diff
@@ -410,6 +412,7 @@ async def test_open_template_project_for_edition(
                     request_scheme=request_scheme,
                     simcore_user_agent=UNDEFINED_DEFAULT_SIMCORE_USER_AGENT_VALUE,
                     request_dns=request_dns,
+                    save_state=True,
```

Comment:
Just checking ... is this ok? it has to be hardcoded to True?

Reply:
changed for correctness, but it had no influence due to how it was mocked